test/EventListener/JsonApiProblemExceptionListenerTest.php

```diff
@@ -114,14 +117,10 @@ public function it_uses_an_exception_transformer(): void
 
         $this->exceptionTransformer->accepts(Argument::type(\Exception::class))->willReturn(true);
         $this->exceptionTransformer->transform(Argument::type(\Exception::class))->willReturn($apiProblem->reveal());
+        $this->event->setResponse(new Response(json_encode([]), Response::HTTP_BAD_REQUEST, ['Content-Type' => 'application/problem+json']));
```

Comment:
The listener should be setting this response, not the unit test. So this might lead to false positives.
The original code was validating the content of the response, not setting it.
Validation on the header and content is missing.

Reply:
Fixed, works fine without the setResponse